main.go

```diff
@@ -190,7 +190,8 @@ func (d *trayDaemon) startIPFS() error {
 	d.menuStartStop.SetTooltip("Starting the IPFS daemon")
 	d.menuStartStop.Disable() // while in progress
 
-	cmd := exec.Command(d.ipfsBinary, "daemon")
+// start the daemon and set the flags
+	cmd := exec.Command(d.ipfsBinary, "daemon", "--migrate", "--routing", "dhtclient", "--enable-gc")
```

Comment:
I think `--routing=dhtclient` would be a bit easier to understand here

Reply:
Alright, some programs don't like doing flags like that, wasn't sure about IPFS.